workflow-controller/controller-service/src/main/java/io/hyscale/controller/validator/impl/DockerValidator.java

```diff
@@ -27,7 +27,7 @@
 import io.hyscale.commons.validator.Validator;
 import io.hyscale.controller.activity.ValidatorActivity;
 import io.hyscale.controller.model.WorkflowContext;
-import io.hyscale.controller.util.ImageDetailsUtil;
+import io.hyscale.servicespec.commons.util.ImageUtil;
 
 @Component
 public class DockerValidator implements Validator<WorkflowContext> {
```

Comment:
DockerDaemon Validator ?
You are internally validating against docker daemon right ?